src/Symfony/Component/Routing/Loader/Configurator/Traits/LocalizedRouteTrait.php

```diff
@@ -26,13 +26,13 @@ trait LocalizedRouteTrait
      * Creates one or many routes.
      *
      * @param string|array $path the path, or the localized paths of the route
-     *
-     * @return Route|RouteCollection
      */
-    final protected function createLocalizedRoute(RouteCollection $collection, string $name, $path, string $namePrefix = '', array $prefixes = null)
+    final protected function createLocalizedRoute(RouteCollection $collection, string $name, $path, string $namePrefix = '', array $prefixes = null): RouteCollection
```

Comment:
As this trait is marked as external, is it ok to add a strict return type or should I only update the PHPDoc?